src/main.cpp

```diff
@@ -64,7 +64,11 @@ int main( int argc, char* argv[] )
 
         int nppc = deck.nppc;
         real_t eps0 = deck.eps;
-        real_t Npe  = n0*Lx*Ly*Lz;
+
+        // TODO: this is a hack, remove this and return to old value
+        real_t Npe  = n0*Lx*0.2*Lz;
```

Comment:
We probably want to do this in the user init.

Reply:
I addressed this. Let me know if there's any other changes you would like to see